packages/components-js/tests/e2e/puppeteer/specs/__snapshots__/default-dom.e2e.ts.snap

```diff
@@ -297,6 +297,31 @@ exports[`should have no basic DOM regression for p-link-tile-model-signature 1`]
 </div>"
 `;
 
+exports[`should have no basic DOM regression for p-link-tile-product 1`] = `
+"<div class="root">
+  <div id="header" class="header">
```

Comment:
Why do we need an ID?

Reply:
For aria-labelledby and aria-describedby